clients/fides-js/docs/interfaces/FidesEvent.md

```diff
@@ -118,23 +127,100 @@ Whether the user should be shown the consent experience. Only available on Fides
 
 What consent method (if any) caused this event.
 
-#### extraDetails.servingToggle?
+#### extraDetails.trigger?
```

Comment:
NOTE: I renamed this from servingToggle -> trigger. We haven't released servingToggle yet, so this doesn't affect any public consumers.

I did this to make the `trigger` property more future-proof: we definitely won't _only_ have toggles triggering events.